### PR TITLE
OSDOCS-3825 Azure Accelerated Networking enabled in OCP 4.11

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
[OSDOCS-3825](https://issues.redhat.com//browse/OSDOCS-3825) Azure Accelerated Networking enabled by default in 4.11

Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3825

Link to docs preview:
http://file.rdu.redhat.com/jbrigman/azure-accelnet/release_notes/ocp-4-11-release-notes.html#ocp-4-11-azure-network-accel

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -168,6 +168,12 @@ For more information, see xref:../installing/disconnected_install/installing-mir
 
 For more information, see xref:../installing/installing_azure/enabling-user-managed-encryption-azure.adoc#enabling-user-managed-encryption-azure[Enabling user-managed encryption for Azure].
 
+[id="ocp-4-11-azure-network-accel]
+==== Accelerated Networking for Azure enabled by default
+{product-title} 4.11 on Azure provides accelerated networking for controller and compute nodes. Accelerated networking is now enabled by default with the installation program for IPI installation.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-11-release-notes.adoc:173]: IPI's [forbidden](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#installer-provisioned-infrastructure).

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -168,6 +168,12 @@ For more information, see xref:../installing/disconnected_install/installing-mir
 
 For more information, see xref:../installing/installing_azure/enabling-user-managed-encryption-azure.adoc#enabling-user-managed-encryption-azure[Enabling user-managed encryption for Azure].
 
+[id="ocp-4-11-azure-network-accel]
```
Comment (kalexand-rh) [inline release_notes/ocp-4-11-release-notes.adoc:171]:
```suggestion
[id="ocp-4-11-azure-network-accel"]
```
This might fix your build error.

**Comment by GroceryBoyJr** [comment]
@MayXuQQ or @mgahagan73 may I please get QE review?

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -168,6 +168,12 @@ For more information, see xref:../installing/disconnected_install/installing-mir
 
 For more information, see xref:../installing/installing_azure/enabling-user-managed-encryption-azure.adoc#enabling-user-managed-encryption-azure[Enabling user-managed encryption for Azure].
 
+[id="ocp-4-11-azure-network-accel"]
+==== Accelerated Networking for Azure enabled by default
+{product-title} 4.11 on Azure provides accelerated networking for controller and compute nodes. Accelerated networking is enabled by default for supported instance types in an installer-provisioned infrastructure installation.
```
Comment (patrickdillon) [inline release_notes/ocp-4-11-release-notes.adoc:173]:
```suggestion
{product-title} 4.11 on Azure provides accelerated networking for control plane and compute nodes. Accelerated networking is enabled by default for supported instance types in an installer-provisioned infrastructure installation. 
```

**Review by patrickdillon — COMMENTED** [review]
LGTM with one small nit

**Comment by patrickdillon** [comment]
LGTM

**Comment by MayXuQQ** [comment]
how about change the ref to https://docs.microsoft.com/en-us/azure/virtual-network/accelerated-networking-overview  @patrickdillon @GroceryBoyJr

**Comment by ahardin-rh** [comment]
:rocket:

**Pull request merged**